src/executor/source/executor/nodes/HttpInputNode.ts

```diff
@@ -12,14 +12,7 @@ export interface HttpInputNode {
  * @returns the data to execute the next node
  */
 export const executeHttpInputNode = async ( node: Node<HttpInputNode>, input: DataflowInput ): Promise<any> => {
-  const method = node.data.params.method || 'GET'
-  switch ( method.toUpperCase() ) {
-    case 'GET':
-    case 'DELETE':
-      return input.query
-    case 'POST':
-    case 'PUT':
-    case 'PATCH':
-      return input.body
-  }
+  const dataFieldName = ['GET', 'DELETE']
+    .includes( node.data.params.method?.toUpperCase() || 'GET' ) ? 'query' : 'body'
```

Comment:
Why isn't this on one line?
For me personally this is uglier than the switch statement but i'm fine with it.